python/cuml/tests/test_hdbscan.py

```diff
@@ -504,13 +514,91 @@ def test_all_points_membership_vectors(nrows,
         algorithm="generic",
         prediction_data=True)
 
-    sk_agg.fit(cp.asnumpy(X))
+    sk_agg.fit(X)
 
     cu_membership_vectors = all_points_membership_vectors(cuml_agg)
-    cu_membership_vectors.sort(axis=1)
     sk_membership_vectors = hdbscan.all_points_membership_vectors(
         sk_agg).astype("float32")
-    sk_membership_vectors.sort(axis=1)
+
     assert_all_points_membership_vectors(
         cu_membership_vectors,
         sk_membership_vectors)
+
+
+@pytest.mark.parametrize('nrows', [1000])
+@pytest.mark.parametrize('n_points_to_predict', [500])
+@pytest.mark.parametrize('dataset', dataset_names)
+@pytest.mark.parametrize('min_samples', [15])
+@pytest.mark.parametrize('cluster_selection_epsilon', [0.0])
```

Comment:
Same question as above- why are we limiting the test cases here (no selection epsilon, only testing `eom`, etc..) My main concern is that if we're not enabling all the possible paths through the code with these options, the users are going to find strange bugs and unexpected behaviors when they enable these options. Not testing these paths at all also has the side effect that things can break as the underlying implementation is updated and we won't know about these breakages.